[2024-03-15 18:07:07,135] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_dt_score 2024-03-15T18:06:00+00:00 [queued]>
[2024-03-15 18:07:07,175] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_dt_score 2024-03-15T18:06:00+00:00 [queued]>
[2024-03-15 18:07:07,176] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-15 18:07:07,176] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2024-03-15 18:07:07,176] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-15 18:07:07,192] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): compute_dt_score> on 2024-03-15T18:06:00+00:00
[2024-03-15 18:07:07,203] {standard_task_runner.py:52} INFO - Started process 41923 to run task
[2024-03-15 18:07:07,208] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'compute_dt_score', '2024-03-15T18:06:00+00:00', '--job-id', '3205', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpz8ip23s_', '--error-file', '/tmp/tmphg0685ze']
[2024-03-15 18:07:07,209] {standard_task_runner.py:77} INFO - Job 3205: Subtask compute_dt_score
[2024-03-15 18:07:07,289] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.compute_dt_score 2024-03-15T18:06:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-15 18:07:07,385] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=compute_dt_score
AIRFLOW_CTX_EXECUTION_DATE=2024-03-15T18:06:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-15T18:06:00+00:00
[2024-03-15 18:07:07,987] {python.py:151} INFO - Done. Returned value was: None
[2024-03-15 18:07:08,017] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=compute_dt_score, execution_date=20240315T180600, start_date=20240315T180707, end_date=20240315T180708
[2024-03-15 18:07:08,123] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-15 18:07:08,185] {local_task_job.py:151} INFO - Task exited with return code 0
